### DISCUSSION TITLE
Is the FastAPI documentation versioned?

### DISCUSSION THREAD

**Discussion by bhavaniravi** [fastapi/fastapi#10805] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

We access https://fastapi.tiangolo.com/ for docs, what if I have to refer an older version of the docs? I cannot find an option to switch, and I cannot see an open issue or PR for the same.

Any reference would be super helpful

### Operating System

macOS

### FastAPI Version

-

### Pydantic Version

-

### Python Version

-

**Comment by shashstormer** [comment, marked as answer]
No The documentation is not versioned, It is suggested that you use latest version. The usage is almost same though and the slight difference in usage in 3.6, 3.9, 3.11, 3.12 are pretty well documented on the site
